client/src/components/ProfilePhoto/useStyles.ts

```diff
@@ -0,0 +1,60 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    marginTop: '10rem',
+    marginBottom: '10rem',
+    alignItems: 'center',
+    justifyContent: 'center',
```

Comment:
Currently, these styles rely on `Container` having `display: 'flex'` but this isn't very future-proof as either the implementation of the `Container` component might change or we might replace it for another component. It would be better to explicitly add `display: 'flex'` here.

Reply:
Added  `display:flex`. I will be careful with my future PR.